packages/affiliates/test/e2e/datasets.js

```diff
@@ -173,34 +173,7 @@ describe("Datasets", function() {
       assert(result.length);
     });
   });
-  describe("Synthprices Serializer", function() {
```

Comment:
I removed these tests mainly because they weren't working over the specified time range, and I deemed this ok, since this is more like an E2E test and is not part of CI.